### ISSUE TITLE
How to understand '# K.binary_crossentropy is helpful to avoid exp overflow.'?

### ISSUE THREAD

**Issue by fandulu** [qqwweee/keras-yolo3#160]
Excuse me, may I ask a question, how to understand the '# K.binary_crossentropy is helpful to avoid exp overflow.' in loss function? Thanks for your help.
